src/backend/Aggregator/HttpServices/Extensions/HttpClientExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace HttpServices.Extensions
+{
+    public static class HttpClientExtensions
+    {
+        public static async Task<TResponse> GetAsync<TResponse> (this HttpClient httpClient, string url)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either change this parameter type to 'System.Uri' or provide an overload which takes a 'System.Uri' parameter.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5020178)